python/semantic_kernel/processes/process_end_step.py

```diff
@@ -1,6 +1,14 @@
 # Copyright (c) Microsoft. All rights reserved.
+import sys
 from typing import TYPE_CHECKING, ClassVar
 
+from semantic_kernel.processes.kernel_process.kernel_process_step_state_metadata import KernelProcessStepStateMetadata
+
+if sys.version_info >= (3, 12):
```

Comment:
I would move this to after the regular imports, then you don't have to worry about splitting them